apps/scheduler-worker/src/dispatch-handler.ts

```
@@ -1,27 +1,158 @@
+      .update(executions)
+      .set({
+        state: 'dispatched',
+        status: 'dispatched',
+        leaseOwner: deps.workerId,
+        workerId: deps.workerId,
+        leaseExpiresAt: lease,
+        attempt: nextAttempt,
+        attemptCount: nextAttempt,
+        version: sql`${executions.version} + 1`,
+        updatedAt: now,
+      })
```

ESLint

[error] Direct execution status write detected. Use ExecutionStateService.transition() from `@octo/runtime-state` instead. Raw writes bypass the FSM CAS guard and will cause state corruption.

(octo-local/no-raw-execution-status-write)